### PR TITLE
Added Travis CI yml file

### PR DESCRIPTION
**Pull request by KaneCheshire**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+language: objective-c
+osx_image: xcode7.3
```
Comment (lucianomarisi) [inline .travis.yml]: shouldn't this be `xcode8 `?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+language: objective-c
```
Comment (lucianomarisi) [inline .travis.yml:1]: for swift I think we can use `generic` instead of `objective-c`, maybe it does not matter thought

File: .travis.yml (hunk 1 above)
Comment (KaneCheshire) [reply .travis.yml]: Xcode 8 is Swift 3, which hasn't been merged in yet 😁 I can change it after the other PR is approved

File: .travis.yml (hunk 2 above)
Comment (KaneCheshire) [reply .travis.yml:1]: I was just using the same format as TABSwiftLayout, happy to change it! I'm new to Travis CI

Comment (lucianomarisi) [reply .travis.yml:1]: probably does not matter, either way it's a mac box

**Review by lucianomarisi — APPROVED** [review]

**Pull request merged**